src/image_processor.py

```diff
@@ -0,0 +1,74 @@
+"""
+画像処理に関するクラス.
+
+@author bizyutyu
+"""
+
+import cv2
+import numpy as np
+import os
+
+
+class ImageProcessor:
+    """CSVファイルをJSONファイルに変換するクラス."""
```

Comment:
このコメントは合っていますか？
```suggestion
    """画像の先鋭化を行うクラス."""
```
とか？